src/Parameters/GetMeetingInfoParameters.php

```diff
@@ -20,80 +20,46 @@
 
 /**
  * Class GetMeetingInfoParameters
- * @package BigBlueButton\Parameters
+ *
+ * @method string getMeetingID()
+ * @method GetMeetingInfoParameters setMeetingID(string $id)
  */
 class GetMeetingInfoParameters extends BaseParameters
 {
     /**
      * @var string
      */
-    private $meetingId;
-
-    /**
-     * @var string
-     */
-    private $password;
+    protected $meetingID;
 
     /**
      * GetMeetingInfoParameters constructor.
      *
-     * @param $meetingId
-     * @param $password
+     * @param $meetingID
      */
-    public function __construct($meetingId, $password)
+    public function __construct($meetingID)
```

Comment:
Is this a breaking change it we remove a constructor parameter. Is there a better way to deprecate it?

Reply:
From my point of view, this pr is resulting in breaking changes anyway, because it will change the behaviour.

I doubt that someone is using the password parameter, because it's not required by the api. If you have better idea, I appreciate every suggestion.

Reply:
that's true. Maybe we can create a list of the api changes, so migrating is easier.